jobs/keepalived/templates/bin/post-start.erb

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+	
+set -e # exit immediately if a simple command exits with a non-zero status
+set -u # report the usage of uninitialized variables
+# source properties file here.
+export IS_HA=<%= link("broker").p("ha_enabled") %>
+iaas_name='<%= p('iaas_config.name', nil) %>'
+
+if [[ "$iaas_name" == "gcp" && "$IS_HA" == "true" ]]; then
+  echo "gcp post-start script execution has started"	
+	
+  export AUTHBASEURL="-authbaseurl=https://accounts.google.com/o/oauth2/token"
```

Comment:
I think it is better to have a jobs_properties file for all the exports and source it here.

Reply:
ok.. moving all exports to a job_properties file. Will be sourcing that file and using it.